k8s-manifests/ingress.yml

```
@@ -0,0 +1,26 @@
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: backend-svc
+            port:
+              number: 3500
+      - path: /
+        pathType: Prefix
+        backend:
+         service:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 9

(indentation)